### DISCUSSION TITLE
Add date to version in changelog.

### DISCUSSION THREAD

**Discussion by xAhlen** [advplyr/audiobookshelf#2977] [Ideas]
Hi
Just feature request to include release date in version in the change log, just to make it easier to see if you're somewhat up to date. 
Also clicking the changelog text cloud maybe take you to the github changelog?

![image](https://github.com/advplyr/audiobookshelf/assets/24399180/5b04aa2d-bbc4-4e3e-b8f4-e306ce9bc093)

**Comment by xAhlen** [comment]
bump

**Reply by nichwall** [reply]
The best way to request features is by opening an issue, discussions aren't checked as often.
